application/src/main/java/org/thingsboard/server/service/mail/DefaultMailService.java

```diff
@@ -264,7 +264,11 @@ private void sendMail(TenantId tenantId, CustomerId customerId, TbEmail tbEmail,
                         byte[] bytes = javax.xml.bind.DatatypeConverter.parseBase64Binary(value);
                         String contentType = helper.getFileTypeMap().getContentType(imgId);
                         InputStreamSource iss = () -> new ByteArrayInputStream(bytes);
-                        helper.addInline(imgId, iss, contentType);
+                        if (tbEmail.isAddFileName()){
+                          helper.addInline(imgId, iss, contentType+"; name=\""+imgId+"\";");
+                        } else {
+                          helper.addInline(imgId, iss, contentType);
+                        }
```

Comment:
Please refactor the code like this:
```
if (tbEmail.isAddFileName()) {
  contentType += "; name=\""+imgId+"\";";
}
helper.addInline(imgId, iss, contentType);
```
Also, fix the code formatting.

Reply:
I fixed it, please check it.

Reply:
Sorry, still nor formatted properly. There are missing spaces.

Reply:
I am sorry that I do not understand "missing spaces" clearly...
Your suggestion code is "2-space-tab.
But another code is "4-space-tab"
So I modified your suggestion to "4-space-tab".
Or another part incorrect format of my code?

![PR11132_format_incorrect](https://github.com/thingsboard/thingsboard/assets/53452427/9c585eef-bc61-419f-b70b-d555ce250935)